### DISCUSSION TITLE
I can't run the app on MAC OS via VS CODE

### DISCUSSION THREAD

**Discussion by aksebzeci** [ikhsan3adi/AuctionApp#7] [Q&A] [answered]
Hello,

I have downloaded your app codes. I have done every step in readme file. When I complete steps, I get the error which is "The asset file '.env' doesn't exist.\nTry creating the file or fixing the path to the file.".

When I look in assets folder, it is true that there isn't a folder that is named ".env". When I try to create the folder the mac os doesn't allow. How am i going to fix this error?

By the way I inspected most of codes, nice work. Congrats.

**Comment by ikhsan3adi** [comment]
I have done a new commit on the repository, hope that solves the problem. Sorry for inconvenience.

Try cloning the repository again

**Comment by aksebzeci** [comment]
Hello,

I have updated dio.dart after your recent commit. But the problem still continues.

![image](https://github.com/ikhsan3adi/flutter-auction-app/assets/79276966/b12b354e-e4fc-4847-9e35-ffbda3a3f2d7)

**Reply by ikhsan3adi** [reply, marked as answer]
It's seems like your `.env` file is missing or moved.

Make sure the `.env` file is in your root project directory.

![Screenshot from 2023-07-23 15-53-46](https://github.com/ikhsan3adi/flutter-auction-app/assets/93766033/26e87fe3-0241-4fdd-aa43-b909356b4559)

**Reply by aksebzeci** [reply]
Thanks, I have solved the problem.
